pkgs/by-name/bu/buttermanager/package.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, fetchFromGitHub
+, python3
+, qt5
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "buttermanager";
+  version = "2.5.0";
+
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "egara";
+    repo = "buttermanager";
+    rev = "${version}";
```

String interpolation is not needed when version is the same as rev.
```suggestion
    rev = version;
```